ux-microservice/app.js

```diff
@@ -12,7 +12,7 @@ app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, "public")));
+app.use("/public", express.static(path.join(__dirname, "/public")));
```

Comment:
I don't mind if this is the case, but are you leaving linting until afterwards on this repo?

Reply:
linting not setup on this service yet, just getting the govuk-frontend in place to demo and deploy.

Reply:
Yeah, no worries.